app/assets/stylesheets/reminders.scss

```diff
@@ -8,5 +8,6 @@ span .reminder_info_in_effect {
 }
 
 .set_reminder {
-  background-color: gray;
+  box-shadow: 1px 3px 1px #000000;
+  background-color: #d7e4ed;
```

Comment:
Color literals like `#d7e4ed` should only be used in variable declarations; they should be referred to via variable everywhere else.